src/styles/abstracts/_variables.scss

```
@@ -0,0 +1,68 @@
+@use 'sass:map';
+
+//colors for all the typography
+$iris: #5d50c6;
+$charleston-green: #222831;
+$white: #ffffff;
+$eerie-black: #191825;
+$giants-orange: #ff5722;
+$brilliant-rose: #f85e9f;
+$gray-black: #19182580;
+$black: #000000;
+
+//font family
+$font-family-primary: 'Circular Std';
+$font-family-secondary: 'Inter';
+
+//different screen sizing of media query
+$mobile-max: 767px;
+$tablet-max: 1024px;
+$tablet-min: 768px;
+$desktop-min: 1025px;
+
+//Theme Colors
+$primary-title: $black;
+$secondary-title: $eerie-black;
+$description: $gray-black;
+$sub-heading: $brilliant-rose;
+$text-primary: $black;
+$text-secondary: $white;
+
+$default-font-size: 16;
```

```suggestion
$base-font-size: 16;
```
Also Please move this declaration to top.